src/order-book/quoteAmountsAndCostsUtils.ts

```diff
@@ -11,9 +11,40 @@ export interface QuoteAmountsAndCostsParams {
 
 const ONE_HUNDRED_BPS = BigInt(100 * 100)
 
-export function getQuoteAmountsAndCosts(params: QuoteAmountsAndCostsParams): QuoteAmountsAndCosts {
-  const { orderParams, sellDecimals, buyDecimals, slippagePercentBps } = params
-  const partnerFeeBps = params.partnerFeeBps ?? 0
+export function getQuoteAmountsWithCosts(params: {
+  sellDecimals: number
+  buyDecimals: number
+  orderParams: OrderParameters
+}) {
```

Comment:
Would be great to keep the return type `QuoteAmountsAndCosts` to be sure if we change the function implementation it still will match the type